code_challenges/advent_of_code/repetition_code.py

```diff
@@ -0,0 +1,37 @@
+def get_messages():
+    with open("./code_challenges/advent_of_code/message_input") as message_input:
+        messages = message_input.read().split("\n")
+    return messages
+
+
+def break_up_messages():
+    messages = get_messages()
+    character_breakdown = []
+
+    for message in messages:
+        for i, character in enumerate(message):
+            while len(character_breakdown) < len(messages[0]):
+                character_breakdown.append({})
+            character_breakdown[i][character] = (
+                character_breakdown[i].get(character, 0) + 1
+            )
+    return character_breakdown
```

Comment:
What I had in mind was something like:
```python
    for message in messages:
        character_counts = {}
        for character in message:
            character_counts[character] = character_counts.get(character, 0) + 1
```

But now I see that the number of dictionary is supposed equal the length of a single message. hmmm.  The way you had it before is better.

or consider:
```python
character_breakdown = []
for i in len(messages[0]:
    char_count = {}
    for message in messages:
        character = message[i]
        char_count[character] = char_count[character].get(character, 0) + 1
    character_breakdown.append(char_count)
```